src/requests/typings/function.ts

```diff
@@ -88,61 +90,64 @@ type ScheduledActionActions = 'create' | 'save' | 'delete' | 'execute'
+export type AppEventEntry = {
   entityType: 'Entry'
-} & AppEventFilter<EntryProps, EntryActions>
-export type AppEventAssetFilter = {
+} & AppEventBase<EntryProps, EntryActions>
+export type AppEventAsset = {
   entityType: 'Asset'
-} & AppEventFilter<AssetProps, AssetActions>
-export type AppEventAppInstallationFilter = {
+} & AppEventBase<AssetProps, AssetActions>
+export type AppEventAppInstallation = {
   entityType: 'AppInstallation'
-} & AppEventFilter<AppInstallationProps, AppInstallationActions>
-export type AppEventTaskFilter = {
+} & AppEventBase<AppInstallationProps, AppInstallationActions>
+export type AppEventTask = {
   entityType: 'Task'
-} & AppEventFilter<TaskProps, TaskActions>
-export type AppEventCommentFilter = {
+} & AppEventBase<TaskProps, TaskActions>
+export type AppEventComment = {
   entityType: 'Comment'
-} & AppEventFilter<CommentProps, CommentActions>
-export type AppEventReleaseFilter = {
+} & AppEventBase<CommentProps, CommentActions>
+export type AppEventRelease = {
   entityType: 'Release'
-} & AppEventFilter<ReleaseProps, ReleaseActions>
-export type AppEventReleaseActionFilter = {
+} & AppEventBase<ReleaseProps, ReleaseActions>
+export type AppEventReleaseAction = {
   entityType: 'ReleaseAction'
-} & AppEventFilter<ReleaseActionProps, ReleaseActionActions>
-export type AppEventScheduledActionFilter = {
+} & AppEventBase<ReleaseActionProps, ReleaseActionActions>
+export type AppEventScheduledAction = {
   entityType: 'ScheduledAction'
-} & AppEventFilter<ScheduledActionProps, ScheduledActionActions>
-export type AppEventBulkActionFilter = {
+} & AppEventBase<ScheduledActionProps, ScheduledActionActions>
+export type AppEventBulkAction = {
   entityType: 'BulkAction'
-} & AppEventFilter<BulkActionProps, BulkActionActions>
-export type AppEventTemplateInstallationFilter = {
+} & AppEventBase<BulkActionProps, BulkActionActions>
+export type AppEventTemplateInstallation = {
   entityType: 'TemplateInstallation'
-} & AppEventFilter<EnvironmentTemplateInstallationProps, TemplateInstallationActions>
-export type AppEventFilterRequest =
-  | AppEventEntryFilter
-  | AppEventAssetFilter
-  | AppEventContentTypeFilter
-  | AppEventAppInstallationFilter
-  | AppEventTaskFilter
-  | AppEventCommentFilter
-  | AppEventReleaseFilter
-  | AppEventReleaseActionFilter
-  | AppEventScheduledActionFilter
-  | AppEventBulkActionFilter
-  | AppEventTemplateInstallationFilter
+} & AppEventBase<EnvironmentTemplateInstallationProps, TemplateInstallationActions>
+export type AppEventRequest =
+  | AppEventEntry
+  | AppEventAsset
+  | AppEventContentType
+  | AppEventAppInstallation
+  | AppEventTask
+  | AppEventComment
+  | AppEventRelease
+  | AppEventReleaseAction
+  | AppEventScheduledAction
+  | AppEventBulkAction
+  | AppEventTemplateInstallation
 
 export type AppEventFilterResponse = {
   result: boolean
 }
 
+export type AppEventHandlerResponse = void
+export type AppEventTransformationResponse = void
```

Comment:
same comment as other PR. I think this could actually be something

Reply:
ok it sounds like we are switching these to `any`